performance-tests/bench/report_parser/argument_parser.cpp

```diff
@@ -0,0 +1,136 @@
+#include "argument_parser.h"
+#include <util.h>
+#include <json_conversion.h>
+#include <BenchTypeSupportImpl.h>
+#include <unordered_map>
+
+using namespace Bench;
+
+bool ArgumentParser::parse(int argc, ACE_TCHAR* argv[], OutputType& output_type,
+  OutputFormat& output_format, Report& report, std::ofstream& output_file_stream) {
+  std::string input_file_path;
+  std::string output_file_path;
+
+  if (argc == 1) {
+    show_usage_prompt();
+  } else {
+    try {
+      for (int i = 1; i < argc; i++) {
+        const ACE_TCHAR* argument = argv[i];
+        std::string option_argument = get_option_argument(i, argc, argv);
+
+        if (!ACE_OS::strcmp(argument, ACE_TEXT("--help"))
+          || !ACE_OS::strcmp(argument, ACE_TEXT("-h"))) {
+          std::cout << usage << std::endl
+            << std::endl
```

Comment:
This seems like it duplicates show_usage() defined below... should this just call show_usage()?

Reply:
Done.